docker/docker-compose.dev.yml

```diff
@@ -39,6 +39,7 @@ services:
       - EMAIL_SMTP_SENDER_EMAIL=${EMAIL_SMTP_SENDER_EMAIL}
     volumes:
       - "../:/usr/app"
+      - "../examples/full-jaffle-shop-demo/dbt:/usr/app/dbt"
```

Comment:
enables devs to do:

- edit dbt code in demo project outside of docker
- hit refresh dbt in ui
- see changes in ui

without rebuilding docker image

Reply:
Nice 🎉